package.json

```diff
@@ -103,6 +105,7 @@
     "webpack": "^4.40.2"
   },
   "devDependencies": {
+    "@storybook/addon-docs": "^5.3.17",
```

Comment:
If you're interested: https://github.com/storybookjs/storybook/tree/871a4ed7bacf546f17379451a9449d6a16d3b3ff/addons/docs/react